### PR TITLE
Konstantinos / Why choose deriv prime section

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/partners/deriv-prime/why-deriv-prime/data.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { TString } from 'types/generics'
```
Comment (aswathy-deriv) [inline src/pages/partners/deriv-prime/why-deriv-prime/data.ts]: Please move this file inside `features->pages->partners->deriv-prime->why-deriv-prime->data.ts`

File: src/pages/partners/deriv-prime/why-deriv-prime/why-deriv-prime.module.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
+@use 'features/styles/theme/theme-mixins' as *;
```
Comment (aswathy-deriv) [inline src/pages/partners/deriv-prime/why-deriv-prime/why-deriv-prime.module.scss]: Similarly this file as `features->pages->partners->deriv-prime->why-deriv-prime->why-deriv-prime.module.scss`

File: src/pages/partners/deriv-prime/why-deriv-prime/index.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import React from 'react'
```
Comment (aswathy-deriv) [inline src/pages/partners/deriv-prime/why-deriv-prime/index.tsx]: Similarly `features->pages->partners->deriv-prime->why-deriv-prime->index.tsx`

File: src/pages/partners/deriv-prime/why-deriv-prime/index.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                            </FlexBox.Box>
+                        )
+                    })}
+                </FlexBox.Box>
+            </FlexBox.Box>
+        </FlexBox.Box>
+    )
+}
+
+export default WhyDerivPrime
```
Comment (aswathy-deriv) [inline src/pages/partners/deriv-prime/why-deriv-prime/index.tsx]: This should be exported as export default WithIntl()(WhyDerivPrime)

File: src/pages/partners/deriv-prime/why-deriv-prime/data.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+type CardTypes = {
+    title?: TString
```
Comment (aswathy-deriv) [inline src/pages/partners/deriv-prime/why-deriv-prime/data.ts]: Make a separate file for types inside `features->pages->partners->deriv-prime->why-deriv-prime->types.ts`

**Review by mayuran-deriv — APPROVED** [review]

File: src/features/pages/partners/deriv-prime/why-deriv-prime/data.ts [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import { CardTypes } from './types'
```
Comment (mayuran-deriv) [inline src/features/pages/partners/deriv-prime/why-deriv-prime/data.ts:1]: do we need to import react here ? 
that's the reason for failing ?

**Pull request closed without merging**